source/man/doveadm-fs.1.rst.in

```diff
@@ -75,6 +80,12 @@ fs put
 
 Store data at the path provided.
 
+**-h** *hash*
+  Save provided hash as content hash. Must be either MD5 or SHA256 in hexdigits.
+**-m** *key=value*
+  Set give metadata key to value. Can be provided as many times as needed. Requires
```

Comment:
The first sentence is a bit off i think.

Reply:
what would you write?

Reply:
A rewritten option could be: "Assign value to given metadata key."